doc/source/overview/faq.md

```diff
@@ -0,0 +1,63 @@
+# Frequently Asked Questions
+
+## General troubleshooting
+
+### I'm getting code errors using a method on my model and my data
+
+There can be many reasons why the method does not work. For code exceptions it is a good idea to check the following:
+ - Read the [docstrings](../api/modules.rst) of the method, paying close attention to the [type hints](https://docs.python.org/3/library/typing.html) as most errors are due to misformatted input arguments.
+ - Check that your model signature (its type and expected inputs/outputs) are in the right format. For further details refer to [White-box and black-box models](../overview/white_box_black_box.md).
```

Comment:
I guess this point covers this [issue](https://github.com/SeldonIO/alibi/issues/516). It might be useful to have this as a separate Question that includes the specific description and solution given in that issue.

Reply:
Similarly, another specific formatting issue I've seen people ask about is including the `batch_size` when passing to `AnchorImage`. i.e. instead of `(28, 28, 3)` they pass `(1, 28, 28, 3)`.

Reply:
Good points, will include.

Reply:
I've added another separate question about different data types, linking to the docs about writing wrapper functions and the Github issue. I've also added a point under general troubleshooting about checking expected input types for `explain` methods, with particular care to single instances vs batches.